Source/ZoomNet/Resources/Groups.cs

```diff
@@ -39,6 +40,19 @@ public async Task<bool> AddUserToGroupAsync(string groupId, string userEmail, Ca
 			.AsMessage()
 			.ConfigureAwait(false);
 
-		return response.IsSuccessStatusCode;
+		if (!response.IsSuccessStatusCode)
+		{
+			return Array.Empty<string>();
+		}
+
+		var jsonResponse = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
You code which is essentially the following steps: 
- "read the response as string", 
- followed by "parse the string into a JSON document", 
- followed by "search the JSON document for a node called 'ids'"

can be simplified with: replace `AsMessage()` with `AsJson()` followed by `return  response.GetPropertyValue("ids", string.Empty).Split(',');`